src/test/java/org/jabref/model/entry/BibEntryWithDefaultConstructorTest.java

```diff
@@ -0,0 +1,272 @@
+package org.jabref.model.entry;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import org.jabref.model.entry.field.BibField;
+import org.jabref.model.entry.field.FieldPriority;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.entry.field.UnknownField;
+import org.jabref.model.entry.types.StandardEntryType;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Simple tests for a BibEntry. There are more test classes testing special features.
+ */
+public class BibEntryWithDefaultConstructorTest {
```

Comment:
I would merge this again with the above tests. Seems like an artificial distinction between tests that use default constructor and the ones that specify an type. For example, the test for the empty cite key above is in spirit the same as the tests below for different field values.